### PR TITLE
Simple subclass

### PR DESCRIPTION
**Pull request by mayou36**

### CODE REVIEW

**Comment by apuignav** [comment]
I like where this is going :-)

File: zfit/core/basemodel.py [hunk 1]
```diff
@@ -900,3 +901,47 @@ def __mul__(self, other):
...
+
+    Example:
+
+    .. code:: python
+
+        class MyPDF(zfit.pdf.ZPDF):
+            _PARAMS = ['mu', 'sigma']
+
+            def _unnormalized_pdf(self, x):
+                # mu = self.params['mu']
```
Comment (apuignav) [inline zfit/core/basemodel.py]: Should these be uncommented?

Comment (mayou36) [reply zfit/core/basemodel.py]: Yes, good catch!

**Comment by mayou36** [comment]
Ready from my side

**Comment by apuignav** [comment]
LGTM!

**Pull request merged**